examples/rhdh-cr.yaml

```
@@ -41,7 +41,7 @@ kind: Secret
 metadata:
   name: secrets-rhdh
 stringData:
-  # generated with the command below (from https://janus-idp.io/docs/auth/service-to-service-auth/#setup):
+  # generated with the command below (from https://rhdh.redhat.com/docs/auth/service-to-service-auth/#setup):
```

Correct link is in the upstream Backstage docs: 

```suggestion
  # generated with the command below (from https://backstage.io/docs/auth/service-to-service-auth/#setup):
```